### ISSUE TITLE
What is the effect of KL divergence?

### ISSUE THREAD

**Issue by Morefre** [NVlabs/DiffusionNFT#9]
Dear authors,
In the code implementation, the training loss has an extra KL divergence regularization term that is not documented in the paper. Could you please elaborate on the effect of this additional loss? Is it necessary and will the model collapse without it?

**Comment by ChenDRAG** [comment]
Without KL, models are easy to collapse. This can be solved by adding decay rate (off-policy training), but at the cost of slowing down performance increase. We thus keep KL regularization.
